README.md

```diff
@@ -127,23 +127,24 @@ PayPal messages props are based on PayPalMessages props and you can see them on
 - **applePayDisplayName**: `string`  
    Name of your store.
 
-
 ### GooglePay
+
 Have a look at [Googles specification](https://developers.google.com/pay/api/web/reference/request-objects) for a detailed explanation of the options
```

Comment:
I think it should be Google's . Please check with Hannah or Irina in cause of doubts